lib/CodeGen/CGExpr.cpp

```diff
@@ -1957,6 +1969,13 @@ void CodeGenFunction::emitAsmCallCodeCopy(const CallExpr *CE) {
   CGM.emitMemcpy(Ptr, Code, Builder.CreateZExtOrTrunc(Length, CGM.Int32Ty));
 }
 
+llvm::Value *CodeGenFunction::emitAsmExternalGetCodeSize(const CallExpr *CE) {
+  // keccak256 function
```

Comment:
What do you mean `keccak256 function` here?

Reply:
Sorry, this is a typo.